packages/oid4vci-demo-frontend/public/locales/development/en/translation.json

```diff
@@ -8,6 +8,8 @@
   "credentials_success_right_pane_button_caption": "Go to {{verifierUrlCaption}}",
   "credentials_success_right_pane_paragraph": "Your {{credentialName}} credential has been\r\nsuccessfully issued to you.\r\n\nYou can use it to verify your identity whenever you\r\ninteract with other parties.",
   "credentials_success_right_pane_title": "Congratulations!",
+  "credentials_success_right_pane_verified_account_paragraph": "\r\n\n\nThe Verified {{credentialName}} credential proofs that UnifiedPost\nhas been able to verify that you have access to the  e-mail address you provided.\r\n\nYou can use this credential to Login to your UnifiedPost account.",
+  "credentials_success_right_pane_verified_account_title": "You have received your Verified {{name}} credential",
```

Comment:
result is that it now says: Verified Account credential credential

Reply:
Just followed the pattern that was already in the translation files and in the legal-entity config file, but they seem to be wrong. Fixed following the code from Niels.